src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -441,10 +441,10 @@ def create(children=None):
 
         instance = OMPParallelDirective(children=children)
```

Comment:
pylint says that there are a couple of unused imports in this file.

Reply:
Removed